Sergey_tasks/JS/Bank-project/Bank.js

```diff
@@ -0,0 +1,275 @@
+
+//--Переменные ---------------------------------------------------------------
+const today = new Date();
+let closeActiveUser;
+
+//--Запись в данных массива accounts в localStorage если они не записаны
+function init(){
+   if(JSON.parse(localStorage.getItem("balance")))
+      return init = null;
+   else localStorage.setItem("balance", JSON.stringify(accounts))
+}
+init();
+
+//--Проверка Login и пароль и если TRUE то вывод на экран данных активного клиента
+btnLogin.addEventListener("click", () =>{
+
+   labelDate.textContent = `${today.getDate()}.0${today.getMonth() + 1}.${today.getFullYear()}`;
+   //--Считываем Login и пароль из Input ------------------
+   const clientLogin = inputLoginUsername.value;
+   const clientPin = parseInt(inputLoginPin.value);
+   //-------------------------------------------------------
+   const clientVerify = JSON.parse(localStorage.getItem("balance")).find(elem => elem.login === clientLogin && elem.pin === clientPin);
+   if(clientVerify){
+      //--Записываем  в localStorage Login активного клиента -------------
+      const activeClient = clientVerify.login;
+      localStorage.setItem("activeClient", activeClient);
+   }
+   initClients();
+   closeActiveUser = setInterval(timer, 1000);
+});
+
+//--Перевод средств (Transfer)-------------------------------------------
+btnTransfer.addEventListener("click", () => {
+   const transfer = (JSON.parse(localStorage.getItem("balance")) || []);
+   //--Переводим средства и записываем в Local Storage -----------------
+   transfer.map(elem => {
+      if (elem.login === inputTransferTo.value && elem.login !== localStorage.getItem("activeClient")){
+         elem.currentBalance += parseInt(inputTransferAmount.value);
+         elem.movements.push(parseInt(inputTransferAmount.value));
+         return {...elem,}
+      }
+      if (elem.login === localStorage.getItem("activeClient")){
+         elem.currentBalance -= parseInt(inputTransferAmount.value);
+         elem.movements.push(- parseInt(inputTransferAmount.value));
+         labelBalance.innerHTML = "elem.currentBalance" + "€";
+         return {...elem,}
+      }
+   });
+   inputTransferTo.value = "";
+   inputTransferAmount.value = "";
+
+   localStorage.setItem("balance", JSON.stringify(transfer));
+   initClients();
+});
+
+//--Добавление строк------------------------------------------------------------------------ 
+function initClients(){
+   const balance = JSON.parse(localStorage.getItem("balance")) || [];
+   const clientUser = balance.find(elem => elem.login === localStorage.getItem("activeClient"));
+
+   containerMovements.innerHTML = "";
+   let transferMoney = 0;
+   let incomeMoney = 0;
+   
+   if(clientUser){ 
+      containerApp.style.opacity = "1";
+      labelBalance.innerHTML = clientUser.currentBalance + "€";
+
+      //--Создаем строки и записываем в них историю движения средств------
+      clientUser.movements.forEach(item => {
+         if(item > 0) {
```

Comment:
code in these two statements is almost the same. The main difference is in classnames. I believe we can move it to one generic function